app/services/ingest_ephemera_service.rb

```diff
@@ -0,0 +1,156 @@
+require 'rdf/ntriples'
+require 'vocab/pul_store'
+
+class IngestEphemeraService
+  def initialize(folder_dir, user, project, logger = Logger.new(STDOUT))
+    @folder_dir = folder_dir
+    @user = user
+    @project = project
+    @logger = logger
+    @vocabs = {}
```

Comment:
Can we extract everything not passed in to private methods?

Reply:
👍  — I made everything private except for the main `ingest` method and the `lookup` method that's tested separately.